doc/source/tutorial.rst

```diff
@@ -0,0 +1,284 @@
+Tutorial
+========
+
+To use Ray, you need to understand the following:
+
+- How Ray uses object IDs to represent immutable remote objects.
```

Comment:
Would it make more sense to switch the order of these? Although the explanation of tasks sort of depends on the explanation of objects, we also expect people to interact with objects mainly by launching tasks (rather than `ray.put`).